frontend/src/v5/ui/v4Adapter/overrides/invitationsList.overrides.ts

```diff
@@ -15,15 +15,11 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { EmptyStateInfo } from '@/v4/routes/components/components.styles';
 import { Actions, Footer, Invitation, List } from '@/v4/routes/components/invitationsDialog/invitationsDialog.styles';
 import { css } from 'styled-components';
 import { secondaryButtonStyling } from '../resuableOverrides.styles';
 
 export default css`
-	${EmptyStateInfo} {
```

Comment:
I'm not sure why this might have caused this, so maybe this has been happening for a while
![image](https://github.com/3drepo/3drepo.io/assets/62481250/1e08ff3d-59f6-4f31-b7f6-e6daf26c03db)
When the invite list is empty the modal is transparent